golioth/cli.py

```diff
@@ -617,6 +617,32 @@ async def delete(config, name):
 
         await project.delete_device(name)
 
+@device.command()
+@click.argument('name')
+@click.argument('tag_id')
+@pass_config
+async def tag_add(config, name, tag_id):
```

Comment:
```suggestion
async def add_tag(config, name, tag_id):
```
Python tends to use verb-noun construction for function names. And this function name translates directly into CLI command, where verb-noun is also more common.